.hadolint.yaml

```diff
@@ -0,0 +1,10 @@
+# List of settings and ignore or safelist findings for the hadolint scanner
+
+# For more information on any settings you can specify, see the actions' documentation here
+# https://github.com/hadolint/hadolint#configure
+override:
+  info: 
+    # Casts the apt-get install <package>=<version> finding as info
+    # We have this set since there is no way to specify version for
+    #  build-essentials in the Dockerfile
+    - DL3008 
```

Comment:
super small nit could we add newlines to the end of the config files to avoid the annoying red circle in github
<img width="189" alt="image" src="https://user-images.githubusercontent.com/447859/219431239-8cf60b2a-787a-4627-938b-dce619e86999.png">

Reply:
In latest commit